src/components/Instafeed/Instafeed.scss

```diff
@@ -0,0 +1,19 @@
+@import "styles/colors.scss";
+@import "styles/mixins.scss";
+
+.instafeed {
+  h3 {
+    @include dynamic-text(2.5em);
+    font-weight: 350;
+  }
+  .crt-logo {
+    color: white !important;
+  }
+  .crt-play-icon {
+    display: none;
+  }
+
+  .crt-post-c {
+    padding: 12px;
```

Comment:
Too small for tailwind?

Reply:
It would be fine but this is another case where I don't have access to the actual elements since they're script generated so it's difficult/impossible.

Reply:
Sounds good.